src/client/src/orangehrmBuzzPlugin/util/composable/useBuzzAPIs.ts

```diff
@@ -91,10 +121,62 @@ export default function useBuzzAPIs(http: APIService) {
     });
   };
 
+  const fetchPostLikes = (postId: number): Promise<AxiosResponse> => {
+    return http.request({
+      method: 'GET',
+      url: `api/v2/buzz/shares/${postId}/likes`,
+    });
+  };
+
+  const fetchPosts = (
+    limit: number,
+    offset: number,
+    sortOrder: 'ASC' | 'DESC',
+    sortField:
+      | 'share.createdAtUtc'
+      | 'share.numOfLikes'
+      | 'share.numOfComments',
+  ): Promise<AxiosResponse<PostsResponse>> => {
+    return http.request({
+      method: 'GET',
+      url: 'api/v2/buzz/feed',
+      params: {
+        limit,
+        offset,
+        sortOrder,
+        sortField,
+      },
+    });
+  };
+
+  const updatePostLike = (
+    postId: number,
+    like: boolean,
+  ): Promise<AxiosResponse> => {
+    return http.request({
+      method: like ? 'DELETE' : 'POST',
+      url: `api/v2/buzz/shares/${postId}/likes`,
+    });
+  };
+
+  const updateCommentLike = (
+    commentId: number,
+    like: boolean,
+  ): Promise<AxiosResponse> => {
+    return http.request({
+      method: like ? 'DELETE' : 'POST',
```

Comment:
```suggestion
      method: like ? 'POST' : 'DELETE',
```
if (like == true) POST request
:thinking:

Reply:
If post is `liked` then send `DELETE`
If post is `not like` then send `POST`